actions/feedback-service.js

```diff
@@ -86,7 +86,7 @@ function update(feedbackClass, cloudant, dbName, minrpi, maxrpi, activities) {
           selector: {
             '$and': [
               {
-                'rpi': {
+                'signDiff': {
                 '$lt': minrpi,
```

Comment:
Shouldn't the change be on this line from the parameter value minrpi to signDiff if the call from the frontend uses signDiff as field name now?

Reply:
The discovery function will return a value called max and min rpi, but they are actually signDiff values. But the result stored in cloudant has an rpi field and a signDiff field. So by changing the search term from rpi to signDiff we are just changing the logic to filter on signDiff.

Reply:
The minrpi and maxrpi values are basically minsigndiff and maxsigndiff

Reply:
Are we changing the name from maxrpi and minrpi in a later PR?

Reply:
well actually the class ends on Thursday so I guess there not a ton of need to do so.